### ISSUE TITLE
Iterate on Monthly Recurring Report

### ISSUE THREAD

**Issue by joanneesteban** [department-of-veterans-affairs/va.gov-team#24740]
# Issue Description
Platform Analytics & Insights shares monthly reports for the Office of Management & Budget (OMB), Digital Modernization Council (DMC), and Veterans Experience Office (VEO). These reports are sent to different stakeholders in the beginning of the month, but generally have the same types of metrics. This issue is to consolidate those reports into one.

[Link to](https://github.com/department-of-veterans-affairs/va.gov-team-sensitive/blob/master/teams/vsp/teams/insights-analytics/monthly-reports-how-to.md) how these reports are shared every month.

## Description
Who is this request for?

Chris Johnston (VEO, OMB, & DMC reports)
    
- What type of analytics support are you looking for?
  ~- [ ] one time metrics report (this is not a KPI dashboard, but a temporary or a one time report)~
  ~- [ ] metrics clarification (verification that the metrics you are looking at are being read accurately)~
  ~- [ ] analytics discussion (please share if you're looking for a general conversation, a KPI setting meeting, etc. in the "additional comments" section)~
  - [x] Monthly recurring reports
    

## Timeframe for Analytics Requested
Data should go as far back as possible, but the dashboard itself should include the last 18-24 months

## Analytics to Track Down

> Name: Monthly VA.gov Summary
> Format: Single bar chart (not stacked or grouped) showing past 24 months (or as much as exists up to 24 months)
> Reports, names, and order:
> 
> - Health Care Application Submissions
> - Disability Claim (526) Submissions
> - Claim Status Views
> - Education Benefit Submissions
> - Burial Submissions
> - Page Views
> - Unique Visitors (was: Users)
> - Unique Logins
> - Customer Satisfaction
> - System Availability
> 
>  
> 
> There are a few other existing data points in GA that I think would be worth adding, provided there’s not too much overhead:
> 
> - Facility Detail Views (add after “Page Views”) – Maps to the DE goal: “Veterans and their families can find a single, authoritative source of information”
> - Letters Downloaded (add after “Claim Status”) – Maps to the DE goal: “Logged-in users have a personalized experience, with relevant and time-saving features”
> - Profile Updates (combined all updates, add after “Unique Logins”) – Maps to goal: “Logged-in users can update their personal information easily and instantly”
> - On-site Searches (add after “Facility Views”) – Maps to goal: “Veterans and their families can find a single, authoritative source of information”
>   - This one isn’t in GA, rather it would likely be a new integration with Search.gov. Not even sure this is possible so certainly a lower priority

## Which Product Questions are you trying to answer?
These reports share larger VA.gov statuses with the offices of OMB & VEO, and the Digital Modernization Council. They are used to report transparency for areas of VA.gov that have been successful and need improvement.

## Other Helpful Questions to think through
- Who is your audience?
- Why do you need this data?
- What are you hoping to find?
- What is your hypothesis that you're hoping the data will support?
- What area of the VA are you hoping to assess?
- How frequently do you need this data?

## Link to Metrics
Current monthly reports
- [Digital Modernization Council](https://va-gov.domo.com/page/42290444)
- [Veterans Experience Office](https://va-gov.domo.com/page/730309340)
- [Office of Management and Budget](https://va-gov.domo.com/page/1587068891)

## Additional Comments
Data discovery work is done in #24741 

Example from Chris:
![image](https://user-images.githubusercontent.com/48964695/118317614-47705300-b4c6-11eb-95e4-358debf40aa5.png)

> Please leave the following blank

## Acceptance Criteria
- [ ] The data has been identified and documented in this ticket - items that we don't have access to are noted with _cannot be accessed_
- [ ] The metrics are shared back with users.

## Definition of Done
- [ ] All appropriate issue tagging is completed
- [ ] All AC completed

**Comment by chloedotbrown** [comment]
Hi @chrisj-usds – wanted to give you an update on this report! We're just wrapping up the final QA of the new iteration and will send this out with the other monthly reports next week. However, I wanted to share the [new iteration](https://va-gov.domo.com/page/1314425343) ahead of time in case you had any questions or wanted to weigh in on any aspects of the design. 

Notes about some of these metrics:

- **Letters Downloaded:** Tagging for debt letter downloads was implemented in summer 2020, so data only goes back until August of that year.
- **On-site Searches:** Tagging for header searches only was implemented at the start of 2021, so data only goes back until January. We _may_ be able to figure out a way in the future to approximate this for historical search data before the tag, but for now we're sticking with the data we're certain about.
- **System Availability:** This is the only place where I broke the style convention of single bar charts. Because cumulative uptime is always between 99.5-99.9%, the line chart allows us to shorten the y-axis so small fluctuations are easier to see. Otherwise, you'd be looking at a graph of 24 nearly-identical bars!
